packages/node_modules/@webex/webex-core/src/webex-core.js

```diff
@@ -544,7 +544,8 @@ const WebexCore = AmpState.extend({
         return p;
       })
       .then((...args) => this._uploadPhaseFinalize(options, ...args))
-      .then((res) => res.body);
+      .then((res) => ({...res.body, ...res.headers}))
+      .catch((err) => err);
```

Comment:
Why are you catching this just to return 'err'? Is this not a real error?

Reply:
I think we can just remove the .catch here now that we've added code to handle it.

Reply:
ok yeah instead of re-throwing or something, I'll just remove it.